include/measurement_kit/common/error.hpp

```diff
@@ -58,5 +58,11 @@ class MaybeNotInitializedError : public Error {
     MaybeNotInitializedError() : Error(2, "unknown_failure 2") {}
 };
 
+// Value error
```

Comment:
Here it should be `/// Value error` (i.e. three slashes, such that the comment is picked up by doxygen)